Unity-Technologies/Graphics
# Automatically show custom DoF parameters when quality is set to custom. [Hold]

# Purpose of this PR
https://fogbugz.unity3d.com/f/cases/1240204/
There were two issues with the Post process UI:
- As described in the bug ticket, the custom options were hidden when the user was selecting "Custom", unless the advanced options were enabled. 
- When advanced was selected and the quality was NOT custom, then the information on the disabled UI elements was just misleading (because it reflects serialezed properties that are used only for custom settings and not the values in the HDRP quality presets)

As suggested by Remi, the solution I implemented in this PR is to show the properties that you can customise only when the custom option is selected. 

Keep in mind that after I finish the DoF improvements (at the moment looks more like a rewrite), the UI might/will change again.

## Manual Tests
- Open an HDRP scene (for example the template)
- Add a Post Processing volume override
- Select custom quality ===> The options to customise the quality now appear automatically.
